server/api/src/rsGeneralAdmin.cpp

```diff
@@ -52,6 +53,59 @@ namespace
         return std::find_if(b, e, [](unsigned char _ch) { return ::isspace(_ch); }) != e;
     } // contains_whitespace
 
+     auto throw_if_user_manages_an_irods_server(
+        RsComm& rsComm,
+        const std::string_view _new_user_type,
+        const std::string_view _user_name
+    ) -> void {
+            if ("rodsadmin" == _new_user_type) return;
+
+            std::set<std::string> server_administrators_in_zone;
+
+            BytesBuf* bbuf = nullptr;
+            irods::at_scope_exit free_buf { [&bbuf] { std::free(bbuf); }  };
+
+            if (const auto ec = rsZoneReport(&rsComm, &bbuf); ec < 0 ) {
+                const auto msg = fmt::format("[{}-{}] - Error obtaining zone report", __func__, __LINE__);
```

Comment:
Remove the use of the function name and line number from all error messages.

Take a look at the other `throw_if_*` functions for examples.

For this particular case, we probably want to change the message to something more generic because the use of `rsZoneReport()` is an implementation detail that will likely confuse the user.

Perhaps ... `"The server experienced an unexpected error while processing the request. Please report to the administrator."`